salt/metalk8s/addons/ui/config/metalk8s-shell-ui-config.yaml.j2

```diff
@@ -17,6 +17,9 @@ spec:
 {%- for user in dex.spec.config.staticPasswords | map(attribute='email') %}
     "{{ user }}": [metalk8s:admin]
 {%- endfor %}
+  logo:
+    light: /brand/assets/logo-light.svg
+    dark: /brand/assets/logo-dark.svg
```

Comment:
Is this logo only useful in the navbar?
In this case, it _could_ makes sense to configure it here, but it used to be in the theme configuration. So, at least, we should document this change somewhere (maybe in the CHANGELOG, but I feel this whole multi-UI setup is lacking documentation anyway, so maybe we should create a new topic in the Developer guide).
One other thing which bothers me, is that we define "light" and "dark" variants of the logos, which makes sense likely. But shouldn't this be related to a theme? How can we know that a theme is dark or light? Would it reference such logo variants by name? IMO, this problem is a good reason for keeping the logo path in the theme configuration (and maybe even keep the logo file base64-encoded in the ConfigMap).